Jenkinsfile

```diff
@@ -79,11 +79,11 @@ pipeline {
                     junit 'nosetests_*.xml'
                     cobertura (
                         coberturaReportFile: 'coverage_*.xml',
-                        failNoReports: true,
-                        failUnhealthy: true,
-                        failUnstable: true,
-                        autoUpdateHealth: true,
-                        autoUpdateStability: true,
+                        //failNoReports: true,
+                        //failUnhealthy: true,
+                        //failUnstable: true,
+                        //autoUpdateHealth: true,
+                        //autoUpdateStability: true,
```

Comment:
I suppose we could remove these checks as it seems they aren't really as helpful.